<launch>

    <!-- openVX CNN graph node -->
    <node pkg = "ti_vl" type = "vl" name = "vl" output = "screen" args = "" required = "true">

        <!-- Input topic name to subscribe-->
        <param name = "input_topic" value = "camera/image_raw"/>

        <!-- Output localization image topic name to publish-->
        <param name = "out_image_topic" value = "vis_localize/out_image"/>

        <!-- Output localization pose topic name to publish-->
        <param name = "out_pose_topic" value = "vis_localize/pose"/>

        <!-- Localization offline map topic name to publish-->
        <param name = "map_topic" value = "vis_localize/map"/>

        <!-- Configuration file for the openVX CNN graph -->
        <rosparam file="$(find ti_vl)/config/params.yaml" subst_value="true" />

    </node>

</launch>
